### PR TITLE
Add copyright and license to all ROS2-beta source files

### PR DESCRIPTION
**Pull request by SamerKhshiboun**
- Added pr_check.sh script which checks Intel copyrights, Apache license, line endings and tabs in for all source files of all folders
- Updated main.yml steps order to first run pr_check script before running any build actions

Tracked by LRS-399

### CODE REVIEW

File: scripts/pr_check.sh [hunk 1]
```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+# This script is designed to ensure all relevant header and source files contain correct copyright
```
Comment (maloel) [inline scripts/pr_check.sh]: Can you please update this documentation so it's more comprehensive?

File: scripts/pr_check.sh [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+     exit 0
+fi
+
+
+cd ..
+check_folder . $1
+cd scripts
+
+if [[ ${fixed} -ne 0 ]]; then
+	 echo "Re-running pr_check..."
```
Comment (maloel) [inline scripts/pr_check.sh]: Tabs...

File: scripts/pr_check.sh [hunk 3]
```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+# This script is designed to ensure all relevant header and source files contain correct copyright
+
```
Comment (maloel) [inline scripts/pr_check.sh:11]: LOL no license on the file that checks licenses... :)

File: scripts/pr_check.sh [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+sudo apt-get install dos2unix
+
+ok=0
+fixed=0
+
+
+function check_folder {
+    for filename in $(find $1 -type f \( -iname \*.cpp -o -iname \*.h -o -iname \*.hpp -o -iname \*.js -o -iname \*.bat -o -iname \*.sh -o -iname \*.txt -o -iname \*.py \)); do
+          if [[ "$filename" == *"importRosbag"* ]]; then
+               continue;
```
Comment (maloel) [inline scripts/pr_check.sh]: Please explain

File: .github/workflows/main.yml [hunk 5]
```diff
@@ -89,4 +95,5 @@ jobs:
         cd ${{github.workspace}}/ros2
         source ${{github.workspace}}/.bashrc
         . install/local_setup.bash
-        python3 src/realsense-ros/realsense2_camera/scripts/rs2_test.py non_existent_file 
+        python3 src/realsense-ros/realsense2_camera/scripts/rs2_test.py non_existent_file
+
```
Comment (Nir-Az) [inline .github/workflows/main.yml:99]: Add ending line

Comment (SamerKhshiboun) [reply .github/workflows/main.yml:99]: Weird, I added a one, but still getting this icon/warning here.

Comment (Nir-Az) [reply .github/workflows/main.yml:99]: Maybe because it has spaces in it.

**Pull request merged**